enterprise/api/v1/views/enterprise_group.py

```diff
@@ -157,7 +161,9 @@ def assign_learners(self, request, group_uuid):
             customer = group.enterprise_customer
         except models.EnterpriseGroup.DoesNotExist as exc:
             raise Http404 from exc
-        if requested_emails := request.POST.dict().get('learner_emails'):
+        if requested_emails := request.data.get('learner_emails'):
+            act_by_date = request.data.get('act_by_date')
+            catalog_uuid = request.data.get('catalog_uuid')
```

Comment:
We should do some input validation on these optional query params, you could use a DRF serializer to achieve this. For instance, you'd want `learner_emails` to be a list of strings that all matched some email regex, which DRF serializers make easy by using `EmailField()`.

Reply:
...and also, `learner_emails` is required for _anything_ to get done by this view.